decidim-generators/lib/decidim/generators/app_generator.rb

```diff
@@ -221,8 +225,43 @@ def tweak_spring
         prepend_to_file "config/spring.rb", "require \"decidim/spring\"\n\n"
       end
 
-      def add_ignore_uploads
-        append_file ".gitignore", "\n# Ignore public uploads\npublic/uploads" unless options["skip_git"]
+      def puma_ssl_options
+        return unless options[:dev_ssl]
+
+        append_file "config/puma.rb", <<~CONFIG
+
+          # Development SSL
+          if ENV["DEV_SSL"] && defined?(Bundler) && (dev_gem = Bundler.load.specs.find { |spec| spec.name == "decidim-dev" })
+            cert_dir = ENV.fetch("DEV_SSL_DIR") { "\#{dev_gem.full_gem_path}/lib/decidim/dev/assets" }
+            ssl_bind(
+              "0.0.0.0",
+              ENV.fetch("DEV_SSL_PORT") { 3443 },
+              cert_pem: File.read("\#{cert_dir}/ssl-cert.pem"),
+              key_pem: File.read("\#{cert_dir}/ssl-key.pem")
+            )
+          end
+        CONFIG
+      end
+
+      def modify_gitignore
```

Comment:
Seems like this is unrelated to the original PR (#9570) and was introduced in another one (#9507). Is this intentional?

Reply:
Probably it's not intentional. I think it came from some merge conflict.

It is reverted now.